### DISCUSSION TITLE
ROCM support for ryzen 7600 on linux

### DISCUSSION THREAD

**Discussion by bil-ash** [ROCm/ROCm#2702] [Q&A] [answered]
According to docs available at https://rocm.docs.amd.com/en/latest/release/windows_support.html and https://rocm.docs.amd.com/en/latest/release/gpu_os_support.html , ROCM is supported on ryzen 7600 but only on Windows and not on Linux. Is this a documentation error or there is some other technical reason? (Screenshots of docs pages attached)

![7600 windows](https://github.com/ROCm/ROCm/assets/33216854/f6566127-4cbe-429d-9564-b020a2b79f6f)
![7600 linux](https://github.com/ROCm/ROCm/assets/33216854/1ff59d16-f0b3-43a7-8cd0-7faafc5299a4)

**Comment by taylding-amd** [comment, marked as answer]
Hi @bil-ash, you are correct that the Radeon RX 7600 is not officially supported or optimized for Linux. However, since it is based on the RDNA 3 architecture, it may still functionally work.
